lib/internal/modules/esm/loader.js

```diff
@@ -525,9 +526,12 @@ class ESMLoader {
         .then(({ module }) => module.getNamespace());
     }
 
-    const namespaces = await PromiseAll(new SafeArrayIterator(jobs));
+    const namespaces = await PromisePrototypeCatch(
+      PromiseAll(new SafeArrayIterator(jobs)),
+      (err) => { throw err; }, // Make errors catchable in a custom loader.
```

Comment:
this is the default catch handler - I’m not sure why this has any effect?

Reply:
Now I'm confused: it wasn't working before this, I add it, it's working, I remove it, it's still working.

Reply:
Yeah it's probably something else that changed on your end, this change should have no effect on the catchability of errors.

Reply:
Weird. I can't explain it